Makefile

```diff
@@ -30,7 +30,11 @@ all: init
 	$(MAKE) -C $(BUILD_DIR) all
 
 install: all
-	cmake -P $(BUILD_DIR)/cmake_install.cmake
+	cmake -DCOMPONENT=octf-install -P $(BUILD_DIR)/cmake_install.cmake
```

Comment:
I'd avoid separated installation. It could broke using camke as external component.